src/kt_class.cpp

```diff
@@ -152,12 +161,18 @@ void KtClass::get_signal_list(List<MethodInfo>* p_list) {
     get_member_list(p_list, signal_infos);
 }
 
+const Dictionary KtClass::get_rpc_methods() {
```

Comment:
Is copy needed by api ?

Reply:
same as abovve

Reply:
Same comment as above i suppose? And yeah it's a copy by the api.